src/components/Common/RoundSwitch/RoundSwitch.js

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import "./roundSwitch.less";
+import * as PropTypes from "prop-types";
+
```

Comment:
use function RoundSwitch({isChecked, onChange}) {
   return (
   ...
   );
}

for better debugging in any case.